components/binary_sensor/rdm6300.rst

```diff
@@ -79,7 +79,7 @@ using :ref:`api-homeassistant_tag_scanned_action`.
       # ...
       on_tag:
         then:
-          - homeassistant.tag_scanned: !lambda 'return uint32_to_string(x);'
+          - homeassistant.tag_scanned: !lambda 'return str_sprintf("%u", x);'
```

Comment:
```suggestion
          - homeassistant.tag_scanned: !lambda 'return to_string(x);'
```

More readable IMO

Reply:
Looks good

Reply:
Hi @MarioGK, could you press the apply button on these suggestions? The PR still has the `str_sprintf` in it.